### PR TITLE
S3 t2 create css for depressed key

### PR DESCRIPTION
**Pull request by RichVelo**
changes to script to allow for black keys to light up and changes to note names g3# to g3Sharp

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea
+.DS_Store
```
Comment (cfbuckley14) [inline .gitignore:2]: duplicated

File: styles.css [hunk 2]
```diff
@@ -173,3 +173,9 @@ h2 {
     left: 50%;
     position: relative;
 }
+
+.depressedKey {
+    box-shadow: 5px 20px 5px #64e30b;
+    background-color: #64e30b;
+    /*transform: scale(-20);*/
```
Comment (cfbuckley14) [inline styles.css]: remove line

**Review by cfbuckley14 — CHANGES_REQUESTED** [review]
a couple of tiny changes pls!:-)

File: styles.css (hunk 2 above)
Comment (RichVelo) [reply styles.css]: 👍

**Review by cfbuckley14 — APPROVED** [review]
Yay!!

**Pull request merged**